tests/flow/test_shortest_path.py

```diff
@@ -135,3 +135,11 @@ def test06_restricted_reltypes(self):
         # The longer traversal will be found
         expected_result = [[1], [2], [3], [4]]
         self.env.assertEqual(actual_result.result_set, expected_result)
+
+    def test07_shortestPath_in_filter(self):
```

Comment:
I think we need to cover cases where the path in `shortestPath` contains both bounded and unbounded entities
e.g.
```
shortestPath((a)-[:E|:E2*]->()
shortestPath(()-[:E|:E2*]->(b)
shortestPath(()-[:E|:E2*]->()
```

In addition to new aliases
`shortestPath((z)-[:E|:E2*]->(x)`

Moreover: `shortestPath(({v:1})-[:E|:E2*]->()`

Reply:
done